src/main/java/com/session/service/Main.java

```diff
@@ -2,14 +2,34 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.session.service.client.SessionClient;
-import com.session.service.client.SessionClientImpl;
+import com.session.service.client.SessionsClient;
+import com.session.service.client.SessionsClientImpl;
 import com.session.service.entities.SessionCreated;
+import com.github.onsdigital.logging.v2.DPLogger;
+import com.github.onsdigital.logging.v2.Logger;
+import com.github.onsdigital.logging.v2.LoggerImpl;
+import com.github.onsdigital.logging.v2.LoggingException;
+import com.github.onsdigital.logging.v2.config.Builder;
+import com.github.onsdigital.logging.v2.serializer.JacksonLogSerialiser;
+import com.github.onsdigital.logging.v2.serializer.LogSerialiser;
+import com.github.onsdigital.logging.v2.storage.LogStore;
+import com.github.onsdigital.logging.v2.storage.MDCLogStore;
+
+import static com.session.service.logging.SessionsClient.info;
+import static com.session.service.logging.SessionsClient.fatal;
 
 public class Main {
 
     public static void main(String[] args) {
-        SessionClient sessionClient = new SessionClientImpl("http://localhost:6666", "");
+
```

Comment:
So this class is not required but it is useful for testing out the client while you are developing it. Maybe move it into the test dir so it's not confused with the implementation.